### PR TITLE
Copter-3.6: check aux switch and mode switch inputs are within range

### PR DESCRIPTION
**Pull request by rmackay9**
This PR helps resolve [this issue re the FrSky SBUS receivers](https://github.com/ArduPilot/ardupilot/issues/10139) in Copter-3.6 (equivalent to [this PR on master](https://github.com/ArduPilot/ardupilot/pull/10199)).  The underlying issue is that some channels recover faster than others from a failsafe.  In particular we have seen cases where the lower channels recover first but the upper channels (used for auxiliary switches) recover much later.  Before the higher channels have recovered they report 874 as their value which can cause the vehicle disarming itself in the air if the arm/disarm function has been attached to the aux switch.

This has been tested in SITL.
sitl-test

### CODE REVIEW

File: ArduCopter/switches.cpp [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 //Documentation of Aux Switch Flags:
 struct {
-    uint8_t CH6_flag    : 2; // 0, 1    // ch6 aux switch : 0 is low or false, 1 is center or true, 2 is high
```
Comment (peterbarker) [inline ArduCopter/switches.cpp]: This change looks unrelated?

Comment (rmackay9) [reply ArduCopter/switches.cpp]: @peterbarker, I had to do that actually or the read_3pos_switch wouldn't accept the .CH6_flag as an argument.

Txs for reviewing!

**Comment by rmackay9** [comment]
@peterbarker, do you think this looks OK otherwise?

**Comment by rmackay9** [comment]
@peterbarker. great. thanks!
